draft-ietf-privacypass-architecture.md

```diff
@@ -84,35 +84,45 @@ informative:
 --- abstract
 
 This document specifies the Privacy Pass architecture and requirements for
-its constituent protocols used for constructing privacy-preserving
-authentication mechanisms. It provides recommendations on how the architecture
-should be deployed to ensure the privacy of clients and the security of all
-participating entities.
+its constituent protocols used for authorization based on privacy-preserving
+authentication mechanisms. It describes the conceptual model of Privacy Pass
+and its protocols, its security and privacy goals, practical deployment models,
+and recommendations for each deployment model that helps ensure the desired
+security and privacy goals are fulfilled.
 
 --- middle
 
 # Introduction
 
 Privacy Pass is an architecture for authorization based on privacy-preserving
-authentication mechanisms. Typical approaches for authorizing clients,
-such as through the use of long-term state (cookies), are not privacy-friendly
-since they allow servers to track clients across sessions and interactions.
-Privacy Pass takes a different approach: instead of presenting linkable
-state-carrying information to servers, e.g., a cookie indicating whether
-or not the client is an authorized user or has completed some prior
-challenge, clients present unlinkable proofs that attest to this information.
-These proofs, or tokens, are private in the sense that a given token cannot
-be linked to the protocol interaction where that token was initially issued.
+authentication mechanisms. In other words, relying parties authenticate clients
+in a privacy-preserving way, i.e., without learning any unique, per-client
+information through the authentication protocol, and then make authorization
+decisions on the basis of that authentication suceeding or failing. Possible
+authorization decisions might be to provide clients with read access to a
+particular resource or write access to a particular API.
```

Comment:
I find the use of "API" here to be a bit odd. I would lean towards implying this and just saying that the authorization allows clients to access a resource.

Reply:
Works for me!